pkg/raft/rafttest/node.go

```diff
@@ -49,7 +49,7 @@ func startNode(id raftpb.PeerID, peers []raft.Peer, iface iface) *node {
 	st := raft.NewMemoryStorage()
 	c := &raft.Config{
 		ID:                        id,
-		ElectionTick:              10,
```

Comment:
NB: the tick duration is [5ms](https://github.com/cockroachdb/cockroach/blob/fc852a20fa1bc479baf39b739506b62a38bc31aa/pkg/raft/rafttest/node.go#L153), so this increase is from 50ms to 250ms (could fix the commit description).